src/plugins/explore/public/components/query_panel/components/footer/date_time_selector.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// TODO: Move this to  already configured datetime range picker
+// This component will be fully functional once integrated with query services.
+
+import React, { useState } from 'react';
+import { EuiSuperDatePicker } from '@elastic/eui';
+
+export const DateTimeRangePicker: React.FC = () => {
```

Comment:
nit. rename the file or component name `DateTimeSelector` to make them consistent?